src/patient-registration/input/custom-input/relationship-input.component .tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState, useEffect, Fragment } from 'react';
+import styles from './../input.css';
+import { SelectInput } from './select-input.component';
+import { getAllRelationshipTypes, getPerson } from '../../patient-registration.resource';
+import { createErrorHandler } from '@openmrs/esm-error-handling';
+import { FieldArray } from 'formik';
+import Autosuggest from 'react-autosuggest';
+//import styles from './autosuggest.css';
+
+interface RelationshipInputProps {
+  setFieldValue(field: string, value: any, shouldValidate?: boolean): void;
+}
+
+export const RelationshipInput: React.FC<RelationshipInputProps> = ({ setFieldValue }) => {
+  const [relationshipTypesOptions, setRelationshipTypesOptions] = useState([]);
+
+  useEffect(() => {
+    const abortController = new AbortController();
+    getAllRelationshipTypes(abortController).then(({ data }) => {
+      data.results.map(option => {
+        setRelationshipTypesOptions(relationshipTypesOptions => [
+          ...relationshipTypesOptions,
+          { uuid: option.uuid + '-A', display: option.aIsToB },
+        ]);
+        setRelationshipTypesOptions(relationshipTypesOptions => [
+          ...relationshipTypesOptions,
+          { uuid: option.uuid + '-B', display: option.bIsToA },
+        ]);
+      });
+    }, createErrorHandler());
+    return () => abortController.abort();
+  }, []);
+
+  const [suggestions, setSuggestions] = useState([]);
+  const [personName, setPersonName] = useState([]);
```

Comment:
Why is there an array called `personName`? What's that supposed to mean? Is this supposed to be `personNames`?